src/elements/common/create-folder-dialog/CreateFolderDialog.scss

```diff
@@ -0,0 +1,3 @@
+.be-modal-input input {
+    width: 100% !important;
```

Comment:
Can this be done without adding an `!important`

Reply:
doesn't seem like it... i tried removing it and the styling doesn't apply.